web/portal/templates/browse.jinja2

```diff
@@ -14,6 +14,7 @@
 
   <p>
     Displaying files in {{target}} <strong>{{description}}</strong>.
+    <a href="{{url_for('render_doc', doc_name=dataset['doc_name'])}}">Learn more about this data set.</a>
```

Comment:
Think this needs to change to 
 <a href="{{url_for('render_doc', doc_name=doc_name)}}">Learn more about this data set.</a>

Reply:
Making that change, and redeployed to the dev portal seems to have fixed that issue with the browse dataset pages

Reply:
Thanks for catching this. Fixed and committed.